MIGRATION_STEPS.md

```diff
@@ -0,0 +1,111 @@
+##Overview
+
+`achambers/ember-cli-deploy`, `LevelBossMike/ember-deploy` and `tedconf/ember-cli-front-end-builds` are coming together to create the offical Ember CLI deployment tool, `ember-cli/ember-cli-deploy`.
+
+Users upgrading from the `ember-cli-deploy` npm package `<= v0.0.6` to `>= v0.4.0` will need to follow the migration steps below as the core codebase of the package will be changing completely.
+
+While we are trying our best to maintain backwards compatability for users of `<= v0.0.6`, this will only be temporary and users are strongly urged to migrate ASAP.
+
+## Migrate config
+
+Migrate your `<= v0.0.6` config from this:
+
+```javascript
+// config/deploy/staging.js
+
+module.exports = {
+  assets: {
+    accessKeyId: 'staging-access-key',
+    secretAccessKey: 'staging-secret',
+    bucket: 'staging-bucket',
+    region: 'eu-west-1'
+  },
+
+  index: {
+    host: 'staging-redis.example.com',
+    port: '1234',
+    password: 'staging-password'
+  }
+};
+```
+
+```javascript
+// config/deploy/production.js
+
+module.exports = {
+  assets: {
+    accessKeyId: 'prod-access-key',
+    secretAccessKey: 'prod-secret',
+    bucket: 'prod-bucket',
+    region: 'eu-west-1'
+  },
+
+  index: {
+    host: 'production-redis.example.com',
+    port: '9876',
+    password: 'prod-password'
+  }
+};
+```
+
+to this:
+
+```javascript
+// config/deploy.js
+
+module.exports = {
+  staging: {
+    buildEnv: 'staging',
+    store: {
+      host: 'staging-redis.example.com',
+      port: 1234
+    },
+    assets: {
+      accessKeyId: 'staging-access-key',
+      secretAccessKey: 'staging-secret',
+      bucket: 'staging-bucket'
+      region: 'eu-west-1'
+    }
+  },
+
+   production: {
+    store: {
+      host: 'production-redis.example.com',
+      port: 9876,
+      password: '<your-redis-secret>'
+    },
+    assets: {
+      accessKeyId: 'prod-access-key',
+      secretAccessKey: 'prod-secret',
+      bucket: 'prod-bucket'
+      region: 'eu-west-1'
+    }
+  }
+};
+```

```

Comment:
Perhaps throw in a line about using environment variables instead of committing secrets.